### PR TITLE
[Docs] Removed authors in utopia-php/messaging docs

### PR DESCRIPTION
**Pull request by mohith2883**
## What does this PR do?

Removes the authors from the following files `composer.json` , `composer.lock` and `Readme.md`

## Test Plan

NA

## Related PRs and Issues
This PR is related to the issue [#4406](https://github.com/appwrite/appwrite/issues/4406)

### Have you read the [Contributing Guidelines on issues](https://github.com/utopia-php/messaging/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by stnguyen90 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -10,12 +10,7 @@
     "lint": "vendor/bin/phpcs",
     "format": "vendor/bin/phpcbf"
   },
-  "authors": [
-    {
-      "name": "Jake Barnby",
-      "email": "jake@appwrite.io"
-    }
-  ],
+
```
Comment (stnguyen90) [inline composer.json:13]: No need for the empty line.

**Review by stnguyen90 — COMMENTED** [review]
Did you manually update the composer.lock file? If so, would you please revert the change? The lock file is automatically generated so it wouldn't really make sense to remove the authors every time we update dependencies.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Please see previous comment

**Comment by mohith2883** [comment]
> Please see previous comment

Hey @stnguyen90 I committed the changes is there anymore changes

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**